nvflare/private/fed/app/simulator/log.config

```diff
@@ -2,20 +2,26 @@
 keys=root
 
 [handlers]
-keys=consoleHandler
+keys=consoleHandler,errorFileHandler
 
 [formatters]
 keys=fullFormatter
 
 [logger_root]
 level=INFO
-handlers=consoleHandler
+handlers=consoleHandler,errorFileHandler
 
 [handler_consoleHandler]
 class=StreamHandler
 level=DEBUG
 formatter=fullFormatter
 args=(sys.stdout,)
 
+[handler_errorFileHandler]
+class=FileHandler
+level=ERROR
+formatter=fullFormatter
+args=('error.log', 'a')
```

Comment:
Are we using this arg?

Reply:
Currently not

Reply:
Got it.

I would suggest either removing it in this PR or submit a follow up PR that makes use of this arg.

Thanks!

Reply:
Actually, I tried removing it and there are errors, so it seems that it does go through some code that checks that there is at least one required positional argument so we may need to keep this for now.

Reply:
You may be able to use it as template: use the configured file name instead of hard-coding it to error.log in fed_utils.  But this may create other complications later when you try to send this log file to the server - you won't know the file name any more unless you keep it somewhere (e.g. data bus).

Let's deal with this in another PR.